src/helm/benchmark/run_specs.py

```diff
@@ -2286,6 +2294,17 @@ def get_cleva_spec(task: str, version: str, subtask: str = None, method: str = A
             max_tokens=150,
         )
         metric_specs = get_basic_metric_specs(["chinese_bleu_1"]) + get_generative_harms_metric_specs()
+    elif task in ["translation"]:
+        adapter_spec = get_generation_adapter_spec(
+            instructions=prompt_setting.instructions,
+            input_noun=prompt_setting.input_noun,
+            newline_after_input_noun=prompt_setting.newline_after_input_noun,
+            output_noun=prompt_setting.output_noun,
+            newline_after_output_noun=prompt_setting.newline_after_output_noun,
+            max_train_instances=5,  # limited by the context length
+            max_tokens=200,
+        )
+        metric_specs = get_cleva_machine_translation_metric_specs()
```

Comment:
You should also add `get_generative_harms_metric_specs()` for generation tasks (though it needs to be updated in the future as the bias metrics in HELM are tailored to English).